A Byte of Python
Traceback (most recent call last):
  File "<string>", line 9, in <module>
AttributeError: 'Livro' object has no attribute 'autor'